src/angularComponent/ngcOmnibox.html

```diff
@@ -1 +1,5 @@
-<h1>Hello world!</h1>
+<input type="text"
+    ng-model="omnibox.ngModel"
+    ng-attr-placeholder="{{::omnibox.placeholder || undefined}}"
```

Comment:
Let's not use bind-once here. Maybe the app-maker wants to update the placeholder dynamically later, you never know. We can't make that decision at the library level.

Also, why `|| undefined`? Isn't empty string better? What falsy values are we trying to normalize to `undefined`?